### DISCUSSION TITLE
amd v2.15.14 拉取订阅报错，大佬们救救

### DISCUSSION THREAD

**Discussion by iCouldFly** [whyour/qinglong#1941] [General]
我的青龙最近拉所有订阅都变更了下面的报错，我没有设置过这个 0.0.0.0:1080。大佬们给看看怎么改正或排查

## 开始执行... 2023-05-20 11:45:13

fatal: unable to access 'https://github.com/xxx/xxx.git/': Failed to connect to 0.0.0.0 port 1080 after 0 ms: Couldn't connect to server

开始更新仓库：/ql/data/repo/xxx_xxx_main

更新/ql/data/repo/xxx_xxx_main失败，请检查网络...

## 执行结束... 2023-05-20 11:45:14  耗时 0 秒

**Comment by jiuyaozhuce** [comment]
咋解决的？

**Reply by iCouldFly** [reply]
-_- 之前梯子装多了，全删了重新装一个就好了

**Comment by lanwangchuan** [comment]
只需要在命令行中执行

git config --global --unset http.proxy 
git config --global --unset https.proxy
